pandas/core/arrays/numpy_.py

```diff
@@ -35,52 +35,68 @@ class PandasDtype(ExtensionDtype):
 
     Parameters
     ----------
-    dtype : numpy.dtype
+    dtype : object
+        Object to be converted to a NumPy data type object.
+
+    See Also
+    --------
+    numpy.dtype
     """
 
     _metadata = ("_dtype",)
 
-    def __init__(self, dtype):
-        dtype = np.dtype(dtype)
-        self._dtype = dtype
-        self._type = dtype.type
```

Comment:
@jorisvandenbossche does turning this into properties have perf implications we need to worry about

Reply:
this is for consistency with name that was changed in #31037

Reply:
OK thanks.  That was the perf affect I had in mind, but pre-coffee i had it backwards.